pkg/services/folder/folderimpl/store.go

```diff
@@ -36,4 +36,7 @@ type store interface {
 
 	// GetFolders returns folders with given uids
 	GetFolders(ctx context.Context, orgID int64, uids []string) ([]*folder.Folder, error)
+
+	// GetDescendants returns all descendants of a folder
```

Comment:
nice!  this is the call required for:
https://github.com/grafana/grafana/blob/main/pkg/registry/apis/folders/legacy_storage.go#L91

Will the same semantics of empty string for anchestor_uid mean, give me all folders?

Reply:
no, I don't think it will return anything with empty string (as it is)
but maybe you are looking for [this service](https://github.com/grafana/grafana/blob/main/pkg/services/annotations/annotationsimpl/xorm_store.go#L384) that was recently introduced